pkg/sqlcmd/commands.go

```diff
@@ -397,44 +397,57 @@ type connectData struct {
 }
 
 func connectCommand(s *Sqlcmd, args []string, line uint) error {
+	cmd := &cobra.Command{
+		Use:   "connect",
+		Short: "Connect to a database",
+		Args:  cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {

```

Comment:
The RunE method, is used when the command action may return an error but Run only prints but not return anything. Here for connect command i used RunE , because connecting to the database may fail and we need to be able to handle that error